tests/apps/courses/test_templates_organization_detail.py

```diff
@@ -104,8 +104,9 @@ def test_templates_organization_detail_cms_published_content(self):
             html=True,
         )
 
-        # The published category should be on the page in its published version
-        self.assertContains(
+        # Published category should not be on the page since we only display
+        # them in draft mode
+        self.assertNotContains(
```

Comment:
When i implemented new integration, i fixed org detail template, it should have showed categories only for admin during edition, but instead categories were always showed:

https://github.com/openfun/richie/commit/a5730dfe28e029c0c6408d00c12c836f929e2459#diff-3ecfe46889781473a6973ad16019ba9aR64

And so we were testing against a wrong behavior, since i fixed it, i updated the test to be correct.